paddle/parameter/Argument.h

```diff
@@ -286,6 +286,12 @@ struct Argument {
    sequence has sub-sequence degrades to a sequence.
    */
   void degradeSequence(const Argument& input, bool useGpu);
+
+  /*
+    @brief    convert the ids vector to value as a sparse matrix
+    @param[out]   the output sparse_mat (already allocated)
+  */
+  void idsToSparseMatrix(MatrixPtr sparse_mat);
```

Comment:
This function does not belong to this class.  Making it member of IVector is better. Also name can be : toOneHotSparseMatrix()